src/utils/global.js

```diff
@@ -1136,6 +1136,8 @@ const global = {
       AbnormalExited: '组件异常退出',
       AbnormalRecovery: '恢复正常',
       '': '-',
+      tenant_lack_of_memory: '超过租户限额',
```

Comment:
This is the reason, not the type of operation.

Reply:
Have to repair